data/Nano_2011/data.csv

```diff
@@ -0,0 +1,226 @@
+Eucalyptus mannensis subsp. mannensis,R+,P-
+Eucalyptus minniritchi,R+,P-
+Eucalyptus normantonensis,R+,P-
+Eucalyptus odontocarpa,R+,P-
+Eucalyptus oxymitra,R+,P-
+Eucalyptus pachyphylla,R+,P-
+Eucalyptus sessilis,R+,P-
+Eucalyptus socialis subsp. eucentrica,R+,P-
+Eucalyptus thozetiana,R+,P-
+Eucalyptus trivalvis,R+,P-
+Eucalyptus victrix,R+,P-
+Euphorbia sarcostemmoides,R-,P-
+Exocarpos sparteus,R+,P-
+Ficus brachypoda,R-,P-
+Gastrolobium brevipes,R-,P+
+Gossypium australe,R+,P+
+Gossypium sturtianum var. sturtianum,R+,P+
+Grevillea albiflora,R+,P+
+Grevillea eriostachya,R+,P+
+Grevillea juncifolia,R+,P+
+Grevillea pyramidalis subsp. leucadendron,R+,P+
+Grevillea refracta subsp. refracta,R-,P+
+Grevillea stenobotrya,R-,P+
+Grevillea striata,R+,P-
+Grevillea wickhamii subsp. aprica,R+,P+
+Gyrostemon ramulosus,R-,P-
+Gyrostemon tepperi,R-,P-
+Hakea arborescens,R+,P-
+Hakea chordophylla,R+,P-
+Hakea divaricata,R+,P-
+Hakea eyreana,R+,P-
+Hakea grammatophylla,R+,P-
+Hakea leucoptera subsp. leucoptera,R-,P-
+Hakea lorea subsp. lorea,R+,P-
+Hakea macrocarpa,R+,P-
+Harnieria kempeana subsp. kempeana,R-,P-
+Hibbertia glaberrima,R+,P-
+Hibiscus burtonii,R+,P+
+Hibiscus krichauffianus,R+,P-
+Hibiscus sturtii var. grandiflorus,R+,P+
+Hibiscus sturtii var. sturtii,R+,P+
+Hibiscus sturtii var. truncatus,R+,P-
+Indigofera basedowii,R+,P+
+Indigofera helmsii,R-,P+
+Indigofera leucotricha,R-,P-
+Indigofera sp. MacDonnell Ranges (P.K.Latz 11587) Peter Wilson,R+,P+
+Ipomoea costata,R+,P+
+Isotropis atropurpurea,R+,P-
+Jacksonia odontoclada,R+,P-
+Jasminum calcareum,R+,P-
+Jasminum didymum subsp. lineare,R+,P-
+Keraudrenia nephrosperma,R-,P+
+Keraudrenia velutina,R+,P-
+Lysiana exocarpi subsp. exocarpi,R-,P-
+Lysiana murrayi,R-,P-
+Lysiana spathulata,R-,P-
+Macrozamia macdonnellii,R+,P-
+Maireana triptera,R-,P-
+Marsdenia australis,R+,P-
+Melaleuca lasiandra,R+,P-
+Micromyrtus flaviflora,R+,P-
+Mirbelia viminalis,R-,P+
+Muehlenbeckia florulenta,R+,P-
+Myoporum montanum,R+,P-
+Olearia ferresii,R+,P-
+Olearia stuartii,R+,P-
+Olearia subspicata,R+,P-
+Ozothamnus kempei,R+,P-
+Pandorea doratoxylon,R+,P-
+Petalostigma nummularium,R+,P-
+Petalostylis cassioides,R+,P+
+Pittosporum angustifolium,R-,P-
+Pluchea dunlopii,R+,P-
+Pluchea ferdinandi-muelleri,R+,P-
+Pluchea tetranthera,R+,P-
+Premna acuminata,R+,P-
+Prostanthera sericea,R+,P-
+Prostanthera striatiflora,R+,P-
+Psydrax latifolia,R-,P-
+Psydrax suaveolens,R+,P-
+Rhagodia eremaea,R-,P-
+Rhagodia parabolica,R-,P-
+Rhagodia spinescens,R-,P-
+Rhyncharrhena linearis,R+,P-
+Ricinocarpos gloria-medii,R+,P-
+Rulingia magniflora,R+,P+
+Santalum acuminatum,R-,P-
+Santalum lanceolatum,R+,P-
+Sarcostemma viminale subsp. australe,R-,P-
+Sauropus ramosissimus,R+,P-
+Scaevola spinescens,R+,P-
+Senna artemisioides nothosubsp. artemisioides,R+,P-
+Senna artemisioides nothosubsp. coriacea,R+,P-
+Senna artemisioides nothosubsp. sturtii,R+,P-
+Senna artemisioides subsp. alicia,R+,P-
+Senna artemisioides subsp. filifolia,R+,P-
```

Comment:
I've realised that there are hidden special characters in a series of names here, which I didn't pick up because they look like spaces. But they are a re-working of an `x` for hybrids. I'll fix them on a commit in a minute